applications/openshift/risk-assessment/compliancesuite_exists/rule.yml

```diff
@@ -0,0 +1,48 @@
+prodtype: ocp4
+
+title: Ensure that Compliance Operator is scanning the cluster
+
+description: |-
+  {{{ weblink(link="https://docs.openshift.com/container-platform/4.7/security/compliance_operator/compliance-operator-understanding.html#compliance-operator-understanding",
+              text="The Compliance Operator") }}}
+  scans the hosts and the platform (OCP)
+  configurations for software flaws and improper configurations according
+  to different compliance benchmarks. It uses OpenSCAP as a backend,
+  which is a known and certified tool to do such scans.
+  
+rationale: |-
+  Vulnerability scanning and risk management are important detective controls
+  for all systems, to detect potential flaws and unauthorised access.
+
+identifiers:
+  cce@ocp4: CCE-83697-3
+
+references:
+  nist: AC-17(2),CM-6,CM-6(1),RA-5,RA-5(5)
```

Comment:
omg, I'm an idiot. I was looking at this PR and the FIPS one at the same time and for some reason thought this is the FIPS one which should reference AC-17(2) and already does actually. So this rule doesn't apply to AC-17(2) and the ref should be removed.
The rest looks good, feel free to just merge right away afterwards.